app/views/collections/show.rb

```diff
@@ -31,7 +31,9 @@ def head_meta
     end
 
     def page_content_heading
-      @collection.title
+      mustache[:page_content_heading] ||= begin
+        (@landing_page_title ? @landing_page_title + ' - ' : '') + @collection.title
```

Comment:
We don't want ```@landing_page_title``` because they always start with 'Europeana', ```@collection.title``` should be unique though.